io.catenax.pcf/RELEASE_NOTES.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [5.0.0] 2023-12-11
```

Comment:
you can change the date to 4th Dec if you want MS3 in next weekly.

Reply:
Unfortunately i won't be able to implement the change that you proposed before Dec 04th, so Dec 11th should be ok from my perspective. Would be great if we could achieve MS2 and MS3 then with regards to the upcoming holidays